src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/WebWorkerTest.cs

```diff
@@ -451,7 +453,7 @@ public async Task WaitAssertsOnJSInteropThreads(Executor executor, NamedCall met
                 {
                     exception = ex;
                 }
-
+                Console.WriteLine("WaitAssertsOnJSInteropThreads: ManagedThreadId: " + Environment.CurrentManagedThreadId + " NativeThreadId: " + WebWorkerTestHelper.NativeThreadId);
```

Comment:
would this tell us which and why `Value is null` ? https://github.com/dotnet/runtime/issues/97914

Reply:
ok, we know which exception, we don't know why.

The "why" is likely because the call is executed on different thread than we expected.

Reply:
this could also be the "why"

https://github.com/dotnet/runtime/blob/5eddce62c285c7f38fbbe2d37406fa1beae5c14b/src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/WebWorkerTestBase.cs#L159

This code is swallowing `OperationCanceledException` instead of passing it

Reply:
Added logs to validate if that is the case.